keyboards/hazel/chaz/keymaps/jasonhazel/keymap.json

```diff
@@ -0,0 +1,128 @@
+{
+  "version": 1,
+  "notes": "",
+  "documentation": "",
+  "keyboard": "hazel/chaz",
+  "keymap": "default",
```

Comment:
would you be able to change `"default"` to `"jasonhazel"`, on line 6, so this file matches directory it's located under? 🙂